docs/TRANSLATING.md

```diff
@@ -246,6 +262,39 @@ git push --force # Push the changes to your repo. Be aware that this will overwr
 
 ---
 
+## Adding a new language (checklist)
+
+The following section describes how to add a new language and what to do or wait for.
+Please open an issue on GitHub and introduce yourself first, even if you intend to add the translation files yourself.
+
+After having opened an issue on GitHub:
+1. Fork the Jamulus repository as described above, then install Qt on your maching. (You will need `lrelease` and `lupdate` later on.)
```

Comment:
```suggestion
1. Fork the Jamulus repository as described above, then install Qt on your machine. (You will need `lrelease` and `lupdate` later on.)
```

Reply:
> install Qt

That's rather vague...

Reply:
No. It depends on the OS, so maybe link to COMPILING.md?